plugins/importexport/csv/CSVImportExportPlugin.inc.php

```diff
@@ -82,177 +82,330 @@ function executeCLI($scriptName, &$args) {
+				) = $fields;
 
 				$press = $pressDao->getByPath($pressPath);
 
-				if ($press) {
+				if (!$press) {
+					echo __('plugins.importexport.csv.unknownPress', ['pressPath' => $pressPath]) . "\n";
+					continue;
+				}
 
-					$supportedLocales = $press->getSupportedSubmissionLocales();
-					if (!is_array($supportedLocales) || count($supportedLocales) < 1) $supportedLocales = array($press->getPrimaryLocale());
-					$authorGroup = $userGroupDao->getDefaultByRoleId($press->getId(), ROLE_ID_AUTHOR);
+				$supportedLocales = $press->getSupportedSubmissionLocales();
+				if (!is_array($supportedLocales) || count($supportedLocales) < 1) {
+					$supportedLocales = [$press->getPrimaryLocale()];
+				}
 
-					// we need a Genre for the files.  Assume a key of MANUSCRIPT as a default.
-					$genre = $genreDao->getByKey('MANUSCRIPT', $press->getId());
+				if (!in_array($locale, $supportedLocales)) {
+					echo __('plugins.importexport.csv.unknownLocale', ['locale' => $locale]) . "\n";
+					continue;
+				}
 
-					if (!$genre) {
-						echo __('plugins.importexport.csv.noGenre') . "\n";
-						exit();
-					}
-					if (!$authorGroup) {
-						echo __('plugins.importexport.csv.noAuthorGroup', array('press' => $pressPath)) . "\n";
-						continue;
-					}
-					if (in_array($locale, $supportedLocales)) {
-						$submission = $submissionDao->newDataObject();
-						$submission->setContextId($press->getId());
-						$submission->setUserId($user->getId());
-						$submission->stampLastActivity();
-						$submission->setStatus(STATUS_PUBLISHED);
-						$submission->setWorkType($isEditedVolume == 1?WORK_TYPE_EDITED_VOLUME:WORK_TYPE_AUTHORED_WORK);
-						$submission->setCopyrightNotice($press->getLocalizedSetting('copyrightNotice'), $locale);
-						$submission->setLocale($locale);
-
-						$series = $seriesDao->getByPath($seriesPath, $press->getId());
-						if ($series) {
-							$submission->setSeriesId($series->getId());
+				// we need a Genre for the files.  Assume a key of MANUSCRIPT as a default.
+				$genre = $genreDao->getByKey('MANUSCRIPT', $press->getId());
+				if (!$genre) {
+					echo __('plugins.importexport.csv.noGenre') . "\n";
+					exit();
+				}
+
+				$authorGroup = $userGroupDao->getDefaultByRoleId($press->getId(), ROLE_ID_AUTHOR);
+
+				if (!$authorGroup) {
+					echo __('plugins.importexport.csv.noAuthorGroup', ['press' => $pressPath]) . "\n";
+					continue;
+				}
+
+				$submission = $submissionDao->newDataObject();
+				$submission->setContextId($press->getId());
+				$submission->stampLastActivity();
+				$submission->stampModified();
+				$submission->setStatus(STATUS_PUBLISHED);
+				$submission->setWorkType($isEditedVolume == 1 ? WORK_TYPE_EDITED_VOLUME : WORK_TYPE_AUTHORED_WORK);
+				$submission->setCopyrightNotice($press->getLocalizedSetting('copyrightNotice'), $locale);
+				$submission->setLocale($locale);
+				$submission->setStageId(WORKFLOW_STAGE_ID_PRODUCTION);
+				$submission->setSubmissionProgress('0');
```

Comment:
Should be a number, not a string; FYI, this becomes a string constant when you move to 3.4.0 or newer.